lib/config/optionsconfiguration.go

```diff
@@ -175,3 +177,26 @@ func (opts OptionsConfiguration) MaxFolderConcurrency() int {
 	// of writing is two, 95-percentile at 12 folders.)
 	return 4 // https://xkcd.com/221/
 }
+
+func (opts OptionsConfiguration) MaxConcurrentIncomingRequestKiB() int {
```

Comment:
In practice this will never be less than megabytes, because  minAllowed is MaxBlockSize * 2 which is 32mb, so this should be MiB.

Reply:
We have a bunch of similar values already though and they are all in KiB